lib/signs/utilities/headways.ex

```diff
@@ -56,6 +62,9 @@ defmodule Signs.Utilities.Headways do
   defp get_stop_ids(sign, nil), do: Signs.Utilities.SourceConfig.sign_stop_ids(sign.source_config)
   defp get_stop_ids(sign, config), do: [sign.headway_stop_id || config.stop_id]
 
+  @spec get_stop_ids(list(SourceConfig.source())) :: [String.t()]
+  defp get_stop_ids(sources), do: Enum.map(sources, & &1.stop_id)
```

Comment:
nit: Since this is tiny and single-use, perhaps we should inline it.

Reply:
Fair point! Will make that tweak